### ISSUE TITLE
allow custom .git-stats path as arg

### ISSUE THREAD

**Issue by Flyingmana** [IonicaBizau/git-stats#8]
for the case I want this stats only for specific projects, it would be cool if I can point to a different file via an argument.

**Comment by IonicaBizau** [comment]
That's a good idea. :+1:

**Comment by IonicaBizau** [comment]
Now we can point to a different config file using the `-c` option:

```sh
$ git-stats -c some/config.json
```

This was fixed in the `v2.0.0` branch. This issue will get closed automatically when `2.0.0` is released. :four_leaf_clover:

**Comment by IonicaBizau** [comment]
This is also settable in `~/.git-stats-config.json` now:

```json
{
   "data_path": "path/to/some/data/file.json"
}
```
